src/commands/imageSearch.ts

```diff
@@ -0,0 +1,38 @@
+import Discord from "discord.js";
+import { searchId, googleToken } from "../config.json";
+import fetch from "node-fetch";
+
+/**
+ * Initiates a google image search on specified term and responds with a link to the image
+ * @param {Discord.Messaage} message
+ */
+export function imageSearch(message: Discord.Message): Promise<void> {
+  var searchTerm = message.content.split(" ");
+  searchTerm = searchTerm.splice(2);
+  var formattedsearch = searchTerm.join("%20");
+  var num = Math.floor(Math.random() * 10);
+  var type =
+    message.content.split(" ")[1] == "animate" ? "&imgType=animated" : "";
+  return fetch(
+    "https://www.googleapis.com/customsearch/v1?key=" +
+      googleToken +
+      "&cx=" +
+      searchId +
+      "&searchType=image" +
+      type +
+      "&q=" +
+      formattedsearch
```

Comment:
Not a comment on your code, but good lord I hate that `fetch` has no way of adding query params. Axios did!!